x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_summary_charts_panel/index.tsx

```diff
@@ -47,25 +53,53 @@ export const AlertsSummaryChartsPanel: React.FC<Props> = ({
   runtimeMappings,
   signalIndexName,
   title = i18n.CHARTS_TITLE,
+  isExpanded,
+  setIsExpanded,
+  groupBySelection,
+  setGroupBySelection,
 }: Props) => {
-  const { toggleStatus, setToggleStatus } = useQueryToggle(DETECTIONS_ALERTS_CHARTS_ID);
-  const [querySkip, setQuerySkip] = useState(!toggleStatus);
+  const isAlertsPageChartsEnabled = useIsExperimentalFeatureEnabled('alertsPageChartsEnabled');
 
+  const { toggleStatus, setToggleStatus } = useQueryToggle(DETECTIONS_ALERTS_CHARTS_ID);
+  const [querySkip, setQuerySkip] = useState(
+    isAlertsPageChartsEnabled ? !isExpanded : !toggleStatus
```

Comment:
This code also looks very same to `alerts_count` and `alerts_histogram`.  I understand this was there before and time is not much before feature freeze so in case you think this logic can be extracted to saperate hook, you can tackle it during BC1 after creating an issue.

Reply:
this is a great point, `querySkip` should be extracted out and handled at a higher level same as `toggleStatus`. I also want to revisit why we have it at the first place? seems redundant if skip is just opposite of toggleStatus